run_tests.py

```diff
@@ -273,7 +273,7 @@ def usage():
                 tf_cfg.dbg(0, "D")
             name = disabled["name"]
             reason = disabled["reason"]
-            tf_cfg.dbg(1, 'Disabled test "%s" : %s' % (name, reason))
+            tf_cfg.dbg(2, 'Disabled test "%s" : %s' % (name, reason))
```

Comment:
Was it accidentally included in the commit?

Reply:
No, i think we dont want disabled tests output when run testswith `-nv`